lib/cocoapods/validator.rb

```diff
@@ -722,7 +722,7 @@ def xcodebuild
       case consumer.platform_name
       when :ios
         command += %w(CODE_SIGN_IDENTITY=- -sdk iphonesimulator)
-        command += Fourflusher::SimControl.new.destination('iPhone 4s', deployment_target)
+        command += Fourflusher::SimControl.new.destination('iPhone 5', deployment_target)
```

Comment:
that's a good idea, just need to teach fourflusher about the order of simulators — downside would be needing to update it whenever new simulators are shipped